src/openzaak/components/zaken/api/viewsets.py

```diff
@@ -429,7 +434,9 @@ class ZaakObjectViewSet(
 
 
 class ZaakInformatieObjectViewSet(
+    FailedNotificationMixin,
     NotificationCreateMixin,
+    NotificationDestroyMixin,
```

Comment:
@sergei-maertens this one was missing here, but not on the viewset for `BesluitInformatieObject`, so I added it here as well